### ISSUE TITLE
quick fix places # type: ignore on wrong line.

### ISSUE THREAD

**Issue by MrYutz** [microsoft/pylance-release#3959]
## Environment data

-   Language Server version: 2023.2.20 (pyright cca7588f)
-   OS and version: Linux / MS Devcontainer: mcr.microsoft.com/devcontainers/python:0-3.11
-   Python: 3.11.1
-   Python version from the listed devcontainer above.

## Code Snippet

![Code_N1sTXlqSF4](https://user-images.githubusercontent.com/38189007/219111131-7e6e53ce-423a-4a6c-a588-e71e6f3b77fd.gif)

## Repro Steps
1 - disable all extensions except python, pylance and devcontainers
2 - create python type error like in the video below.
3 - add more than one line of whitespace after the quick fix indicator.
![Code_D6YeS87dyn](https://user-images.githubusercontent.com/38189007/219115107-bc9cb4d0-986d-49be-9c54-1fd253031d1b.gif)

happens on any line in any python file, including a clean build of devcontainer without cache.

## Expected behavior

add `# type ignore` to the current line

## Actual behavior

adds add  `# type ignore` to the next line if there is a line wrap or if there is more than one line of whitespace after the quick fix indicator.

## Logs

```
(58438) [FG] parsing: /workspace/services/db/migrations/env.py (1ms)
(58438) [FG] binding: /workspace/services/db/migrations/env.py (1ms)
(58438) Background analysis message: getDiagnosticsForRange
(58438) Background analysis message: analyze
(58438) [BG(1)] analyzing: /workspace/services/db/migrations/env.py ...
(58438) [BG(1)]   parsing: /workspace/services/db/migrations/env.py (2ms)
(58438) [BG(1)]   binding: /workspace/services/db/migrations/env.py (0ms)
(58438) [BG(1)]   checking: /workspace/services/db/migrations/env.py (51ms)
(58438) [BG(1)] analyzing: /workspace/services/db/migrations/env.py (53ms)
(58438) Background analysis message: getSemanticTokens delta
(58438) [BG(1)] getSemanticTokens delta previousResultId:1676483707222 at /workspace/services/db/migrations/env.py (4ms)
(58438) Background analysis message: resumeAnalysis
(58438) [BG(1)] indexing: /workspace/services/db/migrations/env.py [found 8] (1ms)
(58438) Indexing Done: /workspace/services/db/migrations/env.py
(58438) Background analysis message: getDiagnosticsForRange
(58438) Background analysis message: getDiagnosticsForRange
(58438) Background inlayHint message
```

**Comment by rchiodo** [comment]
Thanks for the issue. I can repro.

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.3.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023311-1-March-2023-PreRelease)
